tools/reviewer/reviewer.proto

```diff
@@ -75,6 +76,9 @@ message User {
   // Projects the user works on
   repeated string project_id = 8;
   repeated Contribution top_contribution = 9;
+  int32 crystals = 10;
+  // list of things people think are worthy of crystals exchange
```

Comment:
Looking good. Once comment - s/list/List
s/X/Y means replace X with Y. The `s` stands for `switch`.